src/DMLQueryBuilder.php

```diff
@@ -54,6 +55,19 @@ public function insertReturningPks(string $table, array|QueryInterface $columns,
         throw new NotSupportedException(__METHOD__ . ' is not supported by Oracle.');
     }
 
+    public function update(
+        string $table,
+        array $columns,
+        array|string|ExpressionInterface $condition,
+        array|string|ExpressionInterface|null $from = null,
+        array &$params = []
+    ): string {
+        /**
+         * Oracle does not support UPDATE ... FROM ...
+         */
+        return parent::update($table, $columns, $condition, null, $params);
```

Comment:
Should we throw "not supported" exception when from is not null?

Reply:
Done.